data_structures/queue/.gitignore

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
You only need a `.gitignore` in the root of your repo. In fact, if a `.gitignore` file is not in the root it will not be used.